### PR TITLE
Refactor buttons on communities of practice and program areas pages

### LINKED ISSUE #2158: Complete refactor/standardization of all buttons within Communities of Practice/Program Areas Page
### Overview
We need to refactor for buttons within the Home page from the recommendations in the CSS standardization spreadsheet (resource below) so that we have a consistent button appearance through the HfLA site.

**Note:** The buttons should have the same length and width after the changes. if it is not, please add CSS styling to make sure they match.

### Action Items
- [x]  Make the 3 recommended edits for the buttons from the spreadsheet below (2 files will be updated)
  

1.  - [x] Update **line 68** of `website/pages/communities-of-practice.html` with HTML from the `Reccomendation` column
2. - [x] Update **line 70** of `website/pages/communities-of-practice.html` with HTML from the `Reccomendation` column
3. - [x] Update **line 87** of `website/pages/program-areas.html` with HTML from the `Reccomendation` column

- [x] Confirm after the recommended edits that the `Action` of each button works as intended within the Communities of Practice/Join Us/Program Areas page
- [x] Confirm the button length/width within the page are the same after changes. If not refactor CSS (this can be accomplished with custom padding for the specific button element or length/width CSS properties)
- [x] Create a Pull Request with the proposed changes to the button

### Resources/Instructions
[Google Sheets - CSS Standardization Buttons](https://docs.google.com/spreadsheets/d/1n3va-1kwGOCTP4JaXpL3TjgvS_2RWDkurb8PyCJSp-0/edit?usp=sharing)
#1921

### PR DESCRIPTION
**Pull request by Zhu-Joseph**
Fixes #2158 

### What changes did you make and why did you make them ?

  - Updated button on **line 69** on communities of practice page ```website/pages/communities-of-practice.html```
  - Updated button on **line 71** on communities of practice page ```website/pages/communities-of-practice.html```
  - Updated button on **line 87** on program areas page ```website/pages/program-areas.html```

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

Before

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
After

</details>

### CODE REVIEW

**Review by adam-abundis — CHANGES_REQUESTED** [review]
@Zhu-Joseph Thank you for your pull request. My apologies for not being clear in the issue. The button should look the same as before. We need to add css to the specific button elements to make sure they match the before picture. Let me know if you have any questions. Thank you. 🌟 🌟 🌟

**Comment by Zhu-Joseph** [comment]
> @Zhu-Joseph Thank you for your pull request. My apologies for not being clear in the issue. The button should look the same as before. We need to add css to the specific button elements to make sure they match the before picture. Let me know if you have any questions. Thank you. 🌟 🌟 🌟

@abuna1985 Hi Adam, I updated the code and made a commit. I did have to update the css to get the buttons to match what they looked like before. Below is an image of the updated code. 

<details >
<summary> Visuals Changed </summary>

<br>

After2
</details>

File: _sass/components/_communities-of-practice.scss [hunk 1]
```diff
@@ -126,3 +126,7 @@
   margin-bottom: 10px;
   font-size: 14px;
 }
+
+.align-center a {
```
Comment (adam-abundis) [inline _sass/components/_communities-of-practice.scss]: @Zhu-Joseph Remove line 130-132 (`.align-center a` styling). We are going to consolidate the styling.

File: _sass/components/_program-areas.scss [hunk 2]
```diff
@@ -163,3 +163,7 @@
 .join-us-footer-paragraphs {
 margin-bottom: 48px;
 }
+
+.join-us-footer-body a {
```
Comment (adam-abundis) [inline _sass/components/_program-areas.scss]: @Zhu-Joseph Remove 167-169 (`.join-us-footer-body` styling). We are going to consolidate the styling.

**Review by adam-abundis — CHANGES_REQUESTED** [review]
@Zhu-Joseph My apologies for the continuous edits. This is my last one. Can you:
1. Remove the scss files changes (comments in Files Changed)
2. Edit the `btn--default` class in `_sass/elements/_buttons.scss`

**From**
https://github.com/hackforla/website/blob/481ab1973aac0acad969a8a32c8857f9e1365751/_sass/elements/_buttons.scss#L40

**To**
`display: inline-flex;`

This will consolidate the styling since all our buttons are links now. As soon as this is updated, I will approve and merge.

Thank you!

**Comment by Zhu-Joseph** [comment]
> @Zhu-Joseph My apologies for the continuous edits. This is my last one. Can you:
> 
> 1. Remove the scss files changes (comments in Files Changed)
> 2. Edit the `btn--default` class in `_sass/elements/_buttons.scss`
> 
> **From**
> 
> https://github.com/hackforla/website/blob/481ab1973aac0acad969a8a32c8857f9e1365751/_sass/elements/_buttons.scss#L40
> 
> **To** `display: inline-flex;`
> 
> This will consolidate the styling since all our buttons are links now. As soon as this is updated, I will approve and merge.
> 
> Thank you!

Hi Adam, 

I made the following changes:
 - Removed commented code
 - Updated line 40 on ```_sass/elements/_buttons.scss``` to ```display: inline-flex;```

Please let me know if there is anything else you would need me to do!

**Review by adam-abundis — APPROVED** [review]
@Zhu-Joseph Thank you again for making another round of changes and for helping us refactor our buttons. I was able to review the changes and everything looks good to me. I am going to approve and merge this. Keep up the great work! ⭐ ⭐ ⭐

**Pull request merged**